### ISSUE TITLE
[FEATURE] Non-formated code detection

### ISSUE THREAD

**Issue by Wokarol** [YilianSource/brackeys-bot#231]
**Why do you want this feature? What is the problem?**
Massive codeblocks without proper tags "```" are not detected and take a lot of space

**Describe the solution you'd like**
Bot should detect if message is long and contains keywords like " : MonoBehaviour" or "public void". it should be then flagged and message should appear with option to paste it or not.
This decision should be handed to Gurus as they take partial care of development channels.
Feature should be exclusive to development channels.

**Comment by AquaXV** [comment]
Talking about code schemes will trigger the bot then? I see this as potentially making more mess because the bot will create messages for fake triggers.

**Comment by Wokarol** [comment]
Good point, but we can create simple regex for simple syntax detection and when talking about code schemes we usually use code blocks

**Comment by AquaXV** [comment]
You do, new people don't. It needs to be userfriendly. And there is no "simple regex" for code detection. Code is complex in terms detection without using a parser for the language because chaning language can result in different syntax and thus requiring a different regex/parser.

**Comment by Wokarol** [comment]
By simple syntax detection I mean something like "((private)|(public)).*(: MonoBehaviour).*{.*}"

**Comment by AquaXV** [comment]
So if I start talking about making methods private in my class this sentence {may/is already} flag(ged)?

**Comment by Wokarol** [comment]
From my experience, code involving ": MonoBehaviour" and "{ }" in almost all (if not all) cases is in codeblock or inline "codeblock" (the one with singular `

**Comment by YilianSource** [comment]
So just to recap - when unformatted code is detected, the bot should prompt an option for a guru to paste it?

**Comment by Wokarol** [comment]
Yes
